tests/oclc_/test_identifier_lookup_coverage_provider.py

```diff
@@ -0,0 +1,247 @@
+
+    def test__multiple_with_real_parser(self):
+        # Testing that calling _multiple actually returns the correct metadata objects.
+
+        provider = IdentifierLookupCoverageProvider(self._default_collection)
+        tree, identifier = self._tree("multi"), self._id("multi")
+        code, owi_data = provider.parser.initial_look_up(self._db, tree)
+        results = provider._multiple(self._db, owi_data, identifier)
+
+        # The document contained two <work> tags and therefore two OWIs, so we
+        # end up with two results.  They should both be Metadata objects, and
+        # should have the same data source and ISBN.
+
+        eq_(len(results), 2)
+        for result in results:
+            eq_(result.__class__.__name__, "Metadata")
+            eq_(result._data_source, "OCLC Classify")
+            eq_(result.identifiers[0], identifier)
+
+        result_1, result_2 = results
+
+        # Result 1:
+        eq_(result_1.identifiers[1], owi_data[0])
+
+        expected_author_info = ("Adams, Douglas", "n80076765", "113230702", ["Author"], {"deathDate": "2001", "birthDate": "1952"})
+        eq_(len(result_1.contributors), 1)
+        author = result_1.contributors[0]
+        author_info = self._get_contributor_info(author)
+        eq_(author_info, expected_author_info)
+
+        self._check_measurements(result_1.measurements, [3786, 112])
+
+        ddc, lcc, fast = self._get_subjects(result_1.subjects)
+        eq_(ddc[0].identifier, "823.914")
+        eq_(lcc[0].identifier, "PR6051.D3352")
+        eq_(len(fast), 5)
+        eq_([x.identifier for x in fast], ['890366', '1075077', '977455', '977550', '923709'])
+
+        # Result 2:
+        eq_(result_2.identifiers[1], owi_data[1])
+
+        eq_(len(result_2.contributors), 2)
+        author_1, author_2 = result_2.contributors
+
+        eq_(self._get_contributor_info(author_1), ("Gaiman, Neil", "n90640849", "103859257", ["Author"], {}))
+        eq_(self._get_contributor_info(author_2), expected_author_info)
+
+        self._check_measurements(result_2.measurements, [2170, 41])
+
+        ddc, lcc, fast = self._get_subjects(result_2.subjects)
+        eq_(ddc[0].identifier, "823.914")
+        eq_(lcc[0].identifier, "PR6051.D3352")
+        eq_(len(fast), 6)
+        eq_([x.identifier for x in fast], ['963836', '1108670', '1075077', '890366', '977455', '977550'])
+
+    def _get_subjects(self, subjects):
+        # Everything in the list of subjects should be a SubjectData object.
+        eq_(len([x for x in subjects if x.__class__.__name__ == "SubjectData"]), len(subjects))
+        # Extract a sublist for each type of classifier.
+        sublists = [[x for x in subjects if x.type == type] for type in ["DDC", "LCC", "FAST"]]
+        # There should always be 1 DDC classification and 1 LCC classification.
+        eq_((len(sublists[0]), len(sublists[1])), (1, 1))
+        return sublists
+
+    def _check_measurements(self, measurements, values):
```

Comment:
We did really thorough checks of the parsed data in the test of the parser itself, so I think spot checks are okay in these tests. That said, the code already exists, so it might be simpler to leave it alone.